src/Microsoft.Framework.FileSystemGlobbing/Internal/Patterns/PatternBuilder.cs

```diff
@@ -14,9 +15,15 @@ public static class PatternBuilder
 
         public static IPattern Build(string pattern)
```

Comment:
What package will be introduced if using `[NotNull]`? I want to keep the dependencies to minimal.